frontend/packages/llama-stack-modular-ui/frontend/src/app/utilities/axios.ts

```diff
@@ -1,4 +1,8 @@
 // eslint-disable-next-line no-restricted-imports
 import axios from 'axios';
 
-export default axios.create();
+export default axios.create({
+  headers: {
+    Authorization: `Bearer ${process.env.AUTH_TOKEN || 'dummy-token'}`,
```

Comment:
I don't think env will have AUTH_TOKEN in browser. Maybe the backend works right now with dummy-token but we'll probably have to figure out how to get actual user auth token from the application context. We had some conversation about this on [slack](https://redhat-internal.slack.com/archives/C07C0447EAV/p1755179645078049) but not sure of the exact way forward yet.